kore/src/Kore/Repl/Parser.hs

```diff
@@ -268,13 +277,13 @@ severity = sDebug <|> sInfo <|> sWarning <|> sError <|> sCritical
     sError    = Logger.Error    <$ literal "error"
     sCritical = Logger.Critical <$ literal "critical"
 
-logScope :: Parser LogScope
-logScope =
-    LogScope . Set.fromList
-        <$$> literal "[" *> many scope <* literal "]"
+parseLogEntries :: Parser (Set Text)
+parseLogEntries =
+    Set.fromList
+        <$$> literal "[" *> many entry <* literal "]"
   where
-      scope =
-          Logger.Scope . Text.pack
+      entry =
```

Comment:
I think that would lead to some parsing ambiguities in the repl. The repl currently uses square brackets as delimiters for the entries, because we need to parse the severity before and the redirect after. Also, since we're parsing entries only inside the brackets, the entry parser doesn't need to be as restrictive (we can have multiple spaces between entries for example).